helm_chart/ibm_storage_enabler_for_containers/templates/k8s-config-configmap.yml

```diff
@@ -2,8 +2,13 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: k8s-config
+  namespace: ubiquity
```

Comment:
namespace is not required, it will be generated by "helm install -namespace", @shay-berman , do we need a default namespace? guess no